spec/rails_helper.rb

```diff
@@ -36,11 +36,10 @@
 end
 
 RSpec.configure do |config|
-
   config.before(:all) do
-  FactoryBot.reload
-  
-# add `FactoryBot` methods
+    FactoryBot.reload
+  end
```

Comment:
could you clean this up a bit
```
# includes first
config.include ...

# hooks later
config.before() {}
```

or move to different files

Reply:
Cleaned